core/modules/qdisp/JobStatus.h

```diff
@@ -51,7 +51,7 @@ namespace qdisp {
 class JobStatus {
```

Comment:
Resource unit was duplicated between JobDescription and JobStatus. No need for 2 copies.